core/config/metrics-to-audits.js

```diff
@@ -27,6 +28,11 @@ const lcpRelevantAudits = [
   'efficient-animated-content',
   'total-byte-weight',
   'lcp-lazy-loaded',
+  'uses-responsive-images',
+  'uses-optimized-images',
+  'modern-image-formats',
+  'offscreen-images',
+  'uses-http2',
```

Comment:
Considering `total-byte-weight` was applicable to LCP I figured that all of the "make image transfer sizes smaller" audits could also be applicable.